src/attachment_loader.rs

```diff
@@ -114,12 +104,6 @@ impl AttachmentLoader {
     c_ptr!(c_attachment_loader, spAttachmentLoader);
 }
 
-impl Clone for AttachmentLoader {
-    fn clone(&self) -> Self {
-        unsafe { AttachmentLoader::new_from_ptr(self.c_ptr()) }
-    }
-}
-
```

Comment:
I think cloning this type is unsafe. The clone will dispose the pointer from the original instance.